docs/book/features/orphan_snapshots.md

```diff
@@ -0,0 +1,71 @@
+## Orphan snapshots detection and clean-up
+From vSphere CSI plugin's perspective, orphan snapshots are vSphere snapshots that were initiated through the vSphere CSI driver but do not have a corresponding VolumeSnapshotContent object in Kubernetes clusters on the vCenter.
+From Velero vSphere plugin's perspective, orphan snapshots are vSphere snapshots whose local deletion is failing after successful upload to the remote repository.
+Velero vSphere plugin performs following steps as part of backup:
+1. Take a snapshot of the volume.
+2. Upload the snapshot to a remote repository.
+3. Delete the snapshot.
+In these steps, deletion of local snapshot could fail, which leaves behind a snapshot. It can be considered as an orphan snapshot.
+
+Orphan snapshot detection algorithm considers snapshot prefix for listing a snapshot as an orphan snapshot. Snapshot prefix is the prefix used in the snapshot description.
```

Comment:
Where can you find the snapshot description?  Will this be displayed in vSphere UI or is this part of the CNS spec?  Since this is a user facing doc, we need to clearly describe what is visible to the user.

Reply:
snapshot description will be available as part of CNS spec or user can check it using GET /volumes/{volumeId}/snapshots API response body. Should I add this info here?

Reply:
Yes, please add that.

Reply:
added these details